lang/python/numpy/Makefile

```diff
@@ -21,7 +21,7 @@ PKG_CPE_ID:=cpe:/a:numpy:numpy
 PKG_CONFIG_DEPENDS:= \
         CONFIG_NUMPY_OPENBLAS_SUPPORT
 
-PKG_BUILD_DEPENDS:=python-cython/host
+PKG_BUILD_DEPENDS:=python-cython/host pyproject-metadata/host
```

Comment:
this will not work (yet)
there is no `pyproject-metadata/host` package;

you could however try to use `HOST_PYTHON3_PACKAGE_BUILD_DEPENDS`;
there are some examples in the `openwrt-21.02` branch (if you grep there);
the infrastructure is still in place to run that logic
but we'd need @jefferyto to provide an opinion on whether to use that (or not)

Reply:
hy @commodo 

actually I wanted to update chython to version 3.0.7 no plans to update this but at the time I thought numpy needed a chython update and it turns out it didn't. if you think this pull request is bad i would close it let the python package maintainer update this

Reply:
@commodo 

https://github.com/openwrt/packages/pull/22961